recipes/repo.rb

```diff
@@ -87,7 +87,7 @@
   case node['platform_family']
   when 'rhel', 'amazon'
     yum_base_url = 'http://public-repo-1.hortonworks.com/HDP'
-    os = if major_platform_version == 5 || hdp_update_version.to_f >= 2.3
+    os = if hdp_update_version.to_f >= 2.3
```

Comment:
wondering if we should just replace this ``if`` block with just ``os =  "centos#{major_platform_version}"``.  This way, on unsupported combinations of os/distro, the user will just get some repo error (or possibly still work for older releases on centos5), rather than us configuring an incorrect repo which will be much harder to troubleshoot.

Reply:
Probably. COOK-2 😉